### DISCUSSION TITLE
Strekla Compile error

### DISCUSSION THREAD

**Discussion by NobHead9** [Security-Onion-Solutions/securityonion#5094] [Unsupported Versions]
I am working with a distributed architecture consisting of 15 forward nodes, 1 manager, and 3 storage/search nodes.  I am consisting seeing strekla compile errors in Kibana (strekla.keyword: compile error).  Regardless of what I attempt it is occurring on all forward nodes, not just one..  What is the best approach of resolving?

**Comment by weslambert** [comment]
Are you able to attach an example? This could mean that a YARA rule is bad.  Are you using any custom rules, or just default YARA rules?

**Comment by NobHead9** [comment]
Hi Wes,

No, I cant do a screenshot, its a closed environment. The compile error 
is identified in Kibana when I am reviewing Strekla alerts. No custom 
yara rules have been entered & planning on removing all rules to see 
what occurs when slowly adding a few back in at a time.
